cmake/external/jemalloc.cmake

```diff
@@ -12,9 +13,9 @@ endif()
 
 ExternalProject_Add(jemalloc
       PREFIX jemalloc
-      URL ${JEMALLOC_URL}
+      GIT_REPOSITORY ${JEMALLOC_URL}
+      GIT_TAG ${JEMALLOC_TAG}
```

Comment:
>GIT_TAG ${JEMALLOC_TAG} [](http://example.com/codeflow?start=6&length=23)

That would simply not work on windows. It does not build on windows.

Reply:
Could you be more specific about the error you have? Duplicated patch or?

Reply:
Or are you saying it won't be able to checkout the tag?

Reply:
In addition, if it's really something Windows/msbuild related, we can have a generator check there as well. But of course it's better to fix that in a cross-platform way.